app/parser/common.py

```diff
@@ -0,0 +1,106 @@
+"""Support for parsing raw data."""
+
+from abc import ABCMeta, abstractmethod
+from ..image.image import Image
+from ..image.color_format import (ColorFormat, PixelFormat, Endianness)
+import numpy
+
+
+def _calc_lcm(x, y):
+    """Calculate least common multiply.
+
+    Keyword arguments:
+        x: first integer
+        y: second integer
+
+    Returns: Least common multiply of both integers.
+    """
+    if x > y:
+        bigger = x
+    else:
+        bigger = y
+
+    while True:
+        if (bigger % x == 0) and (bigger % y == 0):
+            lcm = bigger
+            break
+        bigger += 1
+
+    return lcm
```

Comment:
Since Python 3.9 there's builtin 'math.lcm()'.  You can also write it using 'math.gcd' or simply 'numpy.lcm()'.